core/server/common/src/main/java/alluxio/master/journal/raft/RaftJournalSystem.java

```diff
@@ -401,6 +396,7 @@ private synchronized void initServer() throws IOException {
         .setProperties(properties)
         .setParameters(parameters)
         .build();
+    super.registerMetrics();
```

Comment:
is it OK to reregister at every gainPrimacy() call?

Reply:
![image](https://user-images.githubusercontent.com/42070967/133186950-cd8aca0a-e20f-4b16-a1f6-9b8a2e22f33b.png)
the register strategy in registerMetrics() is ifAbsent, so it doesn't seem to reregister every time.

Reply:
OK. Still, please move this line after `initServer();` call in `RaftJournalSystem.startInternal()` method.